key/src/ed25519/ed25519_private.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 Kodebox, Inc.
```

Comment:
I think this can be renamed to private.rs since it's under the ed25519 module.

Reply:
Ok, that's more reasonable. I changed it.